themes/addons/javascript/addons.js

```diff
@@ -20,14 +20,16 @@ jQuery(function($) {
 
 	$('[data-copies-field]').on('click', function(e) {
 	    var button = $(e.target);
+	    var message = button.siblings('.copy-field__confirmation');
 	    var fields = $('[data-copiable-field=' + button.data('copies-field') + ']');
 
 	    if (fields.length === 1) {
             fields[0].select();
             document.execCommand('copy');
 
-            // Notify the user that the copy succeeded
+            // Notify the user visually / audibly that the copy succeeded
             button.addClass('copy-field__button--triggered');
+            message.text('Copied!');
```

Comment:
Do you need to clear the text again in the timeout handler?

Reply:
In my testing, it breaks the animation and doesn't result in the screenreader speaking the text next time the button is pressed, so I opted to leave it in place.